ui/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -124,6 +129,24 @@ export function RecipientWithAddress({
     );
   }
 
+  let displayName;
```

Comment:
Minor, but could we instead assign to a `const` using a ternary to avoid the uninitialised `let`?

Reply:
Done in #22515